packages/server/assets/workspacesCore/typedefs/workspaces.graphql

```diff
@@ -46,13 +46,12 @@ type WorkspaceMutations {
     @hasScope(scope: "workspace:create")
   delete(workspaceId: String!): Boolean! @hasScope(scope: "workspace:delete")
   update(input: WorkspaceUpdateInput!): Workspace! @hasScope(scope: "workspace:update")
-  """
-  TODO: `@hasWorkspaceRole(role: WORKSPACE_ADMIN)` for role changes
-  """
-  updateRole(input: WorkspaceRoleUpdateInput!): Workspace!
+  updateRole(input: WorkspaceRoleUpdateInput!): Boolean!
     @hasScope(scope: "workspace:update")
-  deleteRole(input: WorkspaceRoleDeleteInput!): Workspace!
+    @hasWorkspaceRole(role: ADMIN)
+  deleteRole(input: WorkspaceRoleDeleteInput!): Boolean!
```

Comment:
I don't expect mocks to be broken (I don't see any new resolvers), but it would be great if you could enable them and validate that workspace queries still work (in Apollo Explorer in localhost:8081/graphql)

Reply:
ty for nudge, will get in the habit of doing this! Went to do in now though and got this:

![image](https://github.com/user-attachments/assets/90d40030-7496-4643-b702-bc05b249bb79)

Will check in tmr about it

Reply:
@cdriesler That just happens when the server is restarting in the BG due to hotreload, so I don't expect this to be an actual issue